### PR TITLE
Optional network and build-tool parameters to Benchmarking scripts

### PR DESCRIPTION
**Pull request by oneEdoubleD**
- Optional `network` argument to `analyse-logs.sh` defaulting to `mainnet` if not supplied
- Optional `build-tool` argument to `benchmark-chain-sync-mainnet.sh` defaulting to `stack` if not supplied

### CODE REVIEW

File: benchmarking/chain-sync/benchmark-chain-sync-mainnet.sh [hunk 1]
```diff
@@ -23,8 +23,17 @@ ulimit -t $CPU_TIME_LIMIT
 
 date --iso-8601=seconds > STARTTIME
 
-NODE="cabal v2-run exe:cardano-node -- "
-NODE="stack --nix exec cardano-node -- "
+if [[ $1 == 'stack' ]]; then
+  NODE="stack --nix exec cardano-node -- "
+  ARGUMENTS=${@:2}
```
Comment (CodiePP) [inline benchmarking/chain-sync/benchmark-chain-sync-mainnet.sh]: 'shift' will shift all arguments one up.

**Comment by ArturWieczorek** [comment]
@CodiePP , @oneEdoubleD - `bors` timed out, I tried to use `bors r+` but it looks like I am missing some permissions to do so. Can you please restart it ?

**Pull request merged**